examples/webgl_shaders_sky.html

```diff
@@ -50,9 +50,9 @@
 				var effectController = {
 					turbidity: 10,
 					rayleigh: 2,
+					luminance: 1,
```

Comment:
> I stumbled upon this one argument thats not in order with the others

On what order are you referring to? Now the parameter order is different compared to the UI (see `guiChanged()`).

Reply:
I refer to the order of the uniforms. I have updated the PR now to how it shoul have been, just moving the uniforms line.